api/environments/identities/traits/tests/test_views.py

```diff
@@ -658,30 +680,6 @@ def setUp(self) -> None:
             identifier="test-user", environment=self.environment
         )
 
-    def test_can_delete_trait(self):
```

Comment:
How come this was removed?

Reply:
Ah, I think it was moved to [here](https://github.com/Flagsmith/flagsmith/pull/1476/files#diff-81dc8d1f7ef5b002c85a17aeb2c2df7bd8192a84f77f0b14030d77cd4e7a4a95R75), right?